apps/home/stylesheets/howitworks.styl

```diff
@@ -0,0 +1,102 @@
+.HomeHowItWorks
+  border-top 1px solid colors-gray-border
+  border-bottom 1px solid colors-gray-border
+
+  &.HomeSection__content
```

Comment:
This selector is pretty odd. You shouldn't be combining BEM style stuff as a modifier.

Reply:
Shall fix